libnavui-dropin/src/main/java/com/mapbox/navigation/dropin/NavigationViewApi.kt

```diff
@@ -112,4 +113,9 @@ abstract class NavigationViewApi {
      * Enable/Disable replay trip session based on simulated locations.
      */
     abstract fun routeReplayEnabled(enabled: Boolean)
+
+    /**
+     * Current instance of a voice instructions player.
+     */
+    abstract val currentVoiceInstructionsPlayer: MapboxVoiceInstructionsPlayer?
```

Comment:
When it's val it's not obvious that the getter is dynamic. Can we use a method instead?